dbms/src/TableFunctions/TableFunctionODBC.cpp

```diff
@@ -17,6 +17,12 @@
 #include <Common/ODBCBridgeHelper.h>
 #include <Core/Defines.h>
 
+#include <Poco/Data/ODBC/ODBCException.h>
+#include <Poco/Data/ODBC/SessionImpl.h>
```

Comment:
Please, remove this headers, they are redundant and fail build:
/usr/include/Poco/Data/ODBC/Unicode.h:33:10: fatal error: sqlext.h: No such file or directory
 #include <sqlext.h>
          ^~~~~~~~~~ 
https://travis-ci.org/yandex/ClickHouse/jobs/419904248

Reply:
Done.